webapp/src/pages/ResponsePage/ResponsePage.js

```diff
@@ -39,21 +40,30 @@ class ResponsePage extends Component {
             applicationFormService.getForEvent(this.props.event.id),
             responsesService.getResponseDetail(this.props.match.params.id, this.props.event.id),
             tagsService.getTagList(this.props.event.id),
-            reviewService.getReviewAssignments(this.props.event.id)
         ]).then(responses => {
             console.log(responses);
             this.setState({
                 eventLanguages: responses[0].event ? Object.keys(responses[0].event.name) : null,
                 applicationForm: responses[1].formSpec,
                 applicationData: responses[2].detail,
                 tagList: responses[3].tags,
-                reviewers: responses[4].reviewers,
-                error: responses[0].error || responses[1].error || responses[2].error || responses[3].error || responses[4].error,
-                isLoading: false
-            }, this.handleData);
-        });
+                reviewers: responses[2].detail.reviewers,
+                error: responses[0].error || responses[1].error || responses[2].error || responses[3].error,
+            }, this.handleData, this.getReviewResponses(responses[2].detail, responses[2].detail.reviewers));
```

Comment:
Hmm this is interesting, adding a second callback separated by a comma. According to the type specification of setState it only takes a single callback function though, so even if this does work, it would be safer to do it in a single callback function like 
```
()=>{this.handledata(); this.getReviewResponse(...) }
```
to protect ourselves from any changes that might break this